WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1072,8 +1072,45 @@ AddreddBubbleRightChild = @roundCorners(0 AddressBubbleBorderRadius AddressBubbl
 AddressBubbleOp = background: @AddressBubbleOpBgNormal@; color: @AppC@; fill: @AppC@;
 AddressBubbleOpActive = background:@AddressBubbleOpBgActive@; 
 TagBubble = @FontSize-normal@ @ActiveCursor@ padding:6px;
-AddressBubbleHolder = max-height:98px; overflow:auto;
+AddressBubbleHolder = max-height:98px; overflow:auto; position:relative;
 AddressBubbleHolderChild = padding: 6px 8px;
+AttachmentBubbleOpBgNormal = @hexToRGBA(TxtC, 80)@
+AttachmentBubbleHeight = 32px
+AttachmentBubble = @AddressBubble@ height: @AttachmentBubbleHeight@; padding-right: 8px; color: @lighten(SecondaryBrandingColor, 30)@; border-radius : 3px;
+AttachmentBubbleOp = background: @AttachmentBubbleOpBgNormal@; fill: @AppC@;
+AttachmentBubbleHolderChild = padding: 6px;
+AttachmentLeftChild = background: @lighten(SecondaryBrandingColor, 10)@;
+AttachmentBubbleBgActive = @lighten(TxtC, 20)@;
+AttachmentBubbleActive = @ActiveCursor@ background:@AttachmentBubbleBgActive@;
+AttachmentContainerBdrNone = border: none;
+BdrWidthNone = border-width: 0px !important;
+AttachmentPlaceholderHintFontSize = font-size:13px
+
+PositionAbsolute = position: absolute
+AttchDragBdrColor = #8dc53e
+AttachmentDragBdr = 3px dashed @AttchDragBdrColor@
+AttchDragBorderRadius = 5px
+AttachmentEditorIframe = border: @AttachmentDragBdr@ !important; border-radius: @AttchDragBorderRadius@ !important; margin-left: 4px !important; margin-top: 6px !important; width: 99% !important;
```

Comment:
Almost most of the properties here are using `important`. Can we somehow remove it?